### PR TITLE
Add install instructions and compatibility with pip 10

### PR DESCRIPTION
**Pull request by paulmelnikow**
This fixes the `setup.py` in this package, though some of the dependencies will also need the fix, hence specifying pip 9 in the install instructions.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM python:2.7.9
+
+WORKDIR /src
+
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends libsuitesparse-dev libboost-dev
+RUN rm -rf /var/lib/apt/lists/*
+
+# chumpy and lace will not install with pip 10.
```
Comment (lillekemiker) [inline Dockerfile]: Chumpy has been fixed. This should no longer be necessary

File: README.md [hunk 2]
```diff
@@ -4,6 +4,28 @@ lace
 A mesh class loaded with useful geometric and analysis functionality.
 
 
+Installation
+------------
+
+Mac OS:
+```sh
+brew update && brew install boost
+pip install pip==9.0.1 numpy==1.13.1
```
Comment (lillekemiker) [inline README.md]: Again, pip 9 should no longer be necessary

File: README.md [hunk 3]
```diff
@@ -4,6 +4,28 @@ lace
...
+pip install lace
+```
+
+Linux:
+```sh
+apt-get install -y --no-install-recommends libsuitesparse-dev libboost-dev
+pip install pip==9.0.1 numpy==1.13.1
```
Comment (lillekemiker) [inline README.md]: and same comment again

**Comment by paulmelnikow** [comment]
I will update! Thanks.

**Comment by paulmelnikow** [comment]
Oops, I just remembered there was more in this PR. Reopening for the moment…

**Pull request closed without merging**